### PR TITLE
XW-4299 | Redesigning video idle screen

### PR DESCRIPTION
**Pull request by bkoval**
@Wikia/x-wing

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -103,6 +103,7 @@ window.wikiaJWPlayer = function (elementId, options, callback) {
 		playerInstance.setup(playerSetup);
 		logger.info('after setup');
 		logger.subscribeToPlayerErrors(playerInstance);
+		wikiaJWPlayerIdleScreen(playerInstance, i18n);
```
Comment (latata) [inline src/index.js]: all custom modification function are exucuted in `loadJWPlayerScript`. How about moving this there as well?

File: src/styles/overrides.scss [hunk 2]
```diff
@@ -61,6 +61,108 @@
...
+		&.jw-breakpoint-6 {
+			@include play-screen(120px, 42px, 36px, 42px, 18px);
+		}
+
+		&.jw-breakpoint-7 {
+			@include play-screen(150px, 60px, 40px, 44px, 24px);
+		}
+
+		.jw-controls {
+			background: linear-gradient(to bottom, rgba(0,0,0,0) 50%,rgba(0,0,0,0.65) 100%);
```
Comment (latata) [inline src/styles/overrides.scss]: `.65`

File: src/idle-screen.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+			minutes  = '0' + minutes;
+		}
+
+		return minutes + ':' + seconds;
+	}
+
+	playerInstance.on('ready', showDuration);
+}
+
+window.wikiaJWPlayerIdleScreen = wikiaJWPlayerIdleScreen;
```
Comment (hakubo) [inline src/idle-screen.js]: \n :)

File: src/styles/overrides.scss [hunk 4]
```diff
@@ -61,6 +61,112 @@
 		}
 	}
 
+	@mixin play-screen($play-icon-size, $padding, $title-font-size, $title-line-height, $duration-font-size) {
```
Comment (hakubo) [inline src/styles/overrides.scss]: change to idle-screen

File: src/idle-screen.js (hunk 3 above)
Comment (bkoval) [reply src/idle-screen.js]: done

File: src/styles/overrides.scss [hunk 5]
```diff
@@ -61,6 +61,112 @@
...
+			top: auto;
+			max-width: calc(100% - 2 * #{$padding} - #{$play-icon-size});
+
+			.wikia-jw-title-duration {
+				color: white;
+				font-size: $duration-font-size;
+				font-weight: 600;
+				letter-spacing: 0.5px;
+				line-height: $duration-font-size;
+				opacity: 0.5;
```
Comment (latata) [inline src/styles/overrides.scss]: `.5`

File: src/styles/overrides.scss [hunk 6]
```diff
@@ -61,6 +61,112 @@
...
+			padding: 0;
+			position: absolute;
+			top: auto;
+			max-width: calc(100% - 2 * #{$padding} - #{$play-icon-size});
+
+			.wikia-jw-title-duration {
+				color: white;
+				font-size: $duration-font-size;
+				font-weight: 600;
+				letter-spacing: 0.5px;
```
Comment (latata) [inline src/styles/overrides.scss]: `.5px`

**Pull request merged**